chia/full_node/mempool.py

```diff
@@ -328,38 +324,60 @@ def at_full_capacity(self, cost: int) -> bool:
 
         return self.total_mempool_cost() + cost > self.mempool_info.max_size_in_cost
 
+    def get_items_spending_coin_ids(self, spent_coin_ids: List[bytes32]) -> List[MempoolItem]:
+        items: List[MempoolItem] = []
+        for coin_ids in chunks(spent_coin_ids, SQLITE_MAX_VARIABLE_NUMBER):
+            args = ",".join(["?"] * len(coin_ids))
+            with self._db_conn:
```

Comment:
I was going to comment on how this context manager seems like it belong outside of the loop. but then I started thinking that the only thing (afaik) the context manager does is to commit when exiting. Since all we do here is reading, maybe we don't even need it. Do you know?

Reply:
I've used the context manager in a bunch of places in here too, where I don't think we need it. Let's punt on changing this until after our mempool PRs land